supertokens_python/recipe/emailpassword/api/signin.py

```diff
@@ -19,11 +19,13 @@
     from supertokens_python.recipe.emailpassword.interfaces import APIOptions, APIInterface
 
 from supertokens_python.exceptions import raise_bad_input_exception
+from supertokens_python.logger import LoggerCodes, debug_logger, info_logger
 
 from .utils import validate_form_fields_or_throw_error
 
 
 async def handle_sign_in_api(api_implementation: APIInterface, api_options: APIOptions):
+    info_logger("Calling SignInAPI")
```

Comment:
A few comments I added on the node PR apply here as well:
- I think the call log should be debug-level and the response info since the response log includes more information and it's more likely to be useful on its own.
- I'd prefer if both of these kinds of logs were done in a single place (supertokens middleware, etc)
- I'd prefer if the API implementation didn't care about the log level of these messages.